### DISCUSSION TITLE
Podman Community Meeting Announcements

### DISCUSSION THREAD

**Discussion by TomSweeneyRedHat** [containers/podman#11635] [General]
Setting up a discussion to allow for the announcements of upcoming Podman Community and Podman Community Cabal Meetings.

**Most recent  Announcement - December 12, 2022**

As we have no subjects as of yet, and I have had a number of people tell me they won't be able to attend due to their holidays starting early or other related holiday gatherings going on, I'm going to cancel the Podman Community Cabal Meeting for Thursday, December 15, 2022, at 11:00 a.m. EST (UTC-5).

The next Podman Community Cabal Meeting will be held on Thursday, January 19, 2023, at 11:00 a.m. EST (UTC-5).  The next Podman Community Meeting will be held on Tuesday, February 7, 2023, at 11:00 a.m. EST (UTC-5).  As a reminder, the Cabal meetings are generally to talk about Design questions, futures, and related items in regards to Podman.  The Community meeting is used for announcements, demos, and things of interest not only for Podman but anything container related.

If you have something that you'd like to present at either meeting, please let me know.  Likewise, if you have seen something interesting in the container world that you'd like to hear more about, please let me know, and I'll see if I can find someone to do a presentation on it.

**Comment by TomSweeneyRedHat** [comment]
Hey All,
The next Podman Community Meeting is next Tues Oct 5, 2021, at 11:00 a.m. EDT (UTC-4). We have a jammed-packed agenda, with a lot of demos, updates on Podman v3.4, Podman on Mac, and more! Agenda here: https://podman.io/community/meeting/agenda/  The video conference link is on the agenda. 

Hope to see you there!

t

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting is in 24 hours! https://podman.io/community/meeting/agenda/

Hope to see you there, free to attend, and link to the video conference there on the agenda.

**Comment by TomSweeneyRedHat** [comment]
Just a quick reminder that the Podman Community Cabal meeting will be happening in about 6 days and 20 hours from now, at 10:00 a.m. EDT (UTC-4) next Thursday, October 21, 2021.  Only one agenda item at the moment to be discussed, happy to take more.  The video conference link is at the top of the agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal Meeting is happening an hour from now.  We will be talking about Netavark, Podman System Monitor for Mac, quadlet, ARM Testing, and more! Video Conference link in the agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
A quick note, due to a number of the core maintainers being involved in ongoing meetings at the current time frame for the Cabal Meetings, we are bumping the start time out by one hour.  Starting with the November 18, 2021, Podman Community Cabal meeting, we will meet at 11:00 a.m. EST (UTC-5).

**Comment by TomSweeneyRedHat** [comment]
It's almost here!  The next Podman Community Meeting is next Tuesday, November 2, 2021, at 11:00 a.m. EDT (UTC-4).  This one really snuck up on me, as I thought we had another week before it happened.  At the moment, we have only one topic, DIY Networking part II, so if you have something you'd like to talk about or demo, please let me know!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is happening in 16.5 hours from now.  We're talking netavark, Mac, rootless networking, buildkit, and more!  Agenda here: https://podman.io/community/meeting/agenda/  #podman #opensource

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal Meeting is happening in just under a week on Thursday, November 18, 2021, at 11:00 a.m. EST (UTC-5). We're talking http://podman.io  redesign, forwarding play kube to httpd, and more! Agenda with video conference: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Just as a reminder, the meeting now starts at 11:00 a.m., previously it started at 10:00 a.m.  Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal meeting will be starting up in just under 24 hours.  Thu Nov 18, 2021 at 11:00 a.m. EST (UTC-4).  Talking podman.io redesign, play kube changes, and more!  Agenda with video conference: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
Pavel Sosin sent me a note that I thought I'd pass along.  Apparently, the "Google Workspace" tends to eat up the announcements for the meetings that I send via the Podman Mailing list.  They're dropped into Categories->Notifications" folder of GMail and they become invisible unless manually added to the "Tasks" on the right-side panel.  

I'm not sure if the same thing is happening or not for other topics.  But, if you're on the Podman mailing list and you don't have an email coming through your gmail at least on a weekly basis, you might check this.

Thanks for the heads up Pavel!

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting will be next Tuesday, December 7, 2021, at 11:00 a.m. EST (UTC-4).  So far, only one topic, a Podman on Windows Demo.  Hit me up if you have a topic!  Video Conference: https://bluejeans.com/880216278/2568 Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal meeting is happening in just under 24 hours. Thursday, December 16, 2021, 11:00 a.m. EST (UTC-4). We'll be talking about Lima and have time for topics of interest. Agenda [here](https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both).

Hope to see you there!

t

**Comment by TomSweeneyRedHat** [comment]
The Podman Community meeting is happening 20 hours from now at 11:00 am EST (UTC-5) Tues Feb 1, 2022. Agenda with conference link here: https://podman.io/community/meeting/agenda/.   Video Conference: https://bluejeans.com/880216278/2568

We'll be seeing demos on the Windows Client, Podman's Desktop, Sparsefile handling, and the new Network stack that's coming in Podman v4.0

Hope to see you there!

t

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal meeting is happening in a little under 18 hours from now at 11:00 a.m. EST (UTC-5) on Thu Feb 17, 2022. Agenda with video conference link [here](https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both). One big planned talk on a new Metapackage that will move a number of related sub-packages into one, and room for other questions if you have them.

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Cabal meeting is happening in 44 hours on Thurs March 17, 2022, at 11:00 am EDT (UTC-5). We'll be talking /etc/hosts in containers, macOS volumes, and room for your topics! Wearing green is optional, but strongly encouraged. The meeting is free, and the video conference link is in the agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community meeting is happening in just under 24 hours, on Tuesday, Apr 5, 2022, at 11:00 a.m. EDT (UTC-5).  The agenda is still a WIP with room for topics.  The Agenda with video link here: https://podman.io/community/meeting/agenda/

**Comment by TomSweeneyRedHat** [comment]
39 hours from now, on Thu April 21, 2022 at 11:00 am EDT (UTC-5), we'll be holding the next Podman Cabal meeting. At the moment only one topic on Contribution policies in GitHub. Happy to take more topics! Agenda with video link here: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is happening in a little under 21 hours from now, at 11:00 am EDT(UTC-5). We'll be talking lock contention, vendoring and release hygiene, and API specgen/create options. Conference link in the agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

    We'll also have some time for topics from the floor, hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is happening this Tuesday, June 7, 2022, @ 11:00 a.m. EDT(UTC-5). We'll be having a new demo on the new Desktop, what is being planned for the next few releases, and more! In fact, we've room for another topic or two, so if you have one that you'd like to present or discuss, drop me a line.

Free to attend here: https://bluejeans.com/880216278/2568

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal Meeting is happening just under 21 hours from now at 11:00 am EDT on Thu July 21, 2022 (UTC-4).  We only have 1 topic about man page cleanups, and welcome more!  
Free to attend here: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting is happening tomorrow, just over 24 hours from now, on Tuesday, August 2, 2022, at 11:00 a.m. EDT(UTC-4).  We're very happy to have @sallyom presenting Fetchit for us, and we're looking for more topics!  Agenda with conference link: https://podman.io/community/meeting/agenda/

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal Meeting scheduled for this Thursday, August 18, 2022, at 11:00 a.m. EDT (UTC-4) has been canceled.  We recently realized that most of the attendees will be attending DevConf.US at that time.  DevConf.US is free to attend, and you can do so in person or virtually.  The only requirement is you need to sign up, and you can do so for free at https://www.devconf.info/us/.  A bunch of info there, too, on the times and topics.  FWIW, Dan Walsh will be doing the Keynote on Saturday, August 20, 2022, at 9:30 a.m. EDT (UTC-4).

    Our next Podman Community Cabal meeting will be on Thursday, September 15, 2022, at 11:00 a.m. EDT (UTC-4).

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal Meeting is happening in just under 24 hours at 11:00am EDT(UTC-4) on Thu September 15, 2022. We'll be talking Kube YAML support, ZSTD update, Confidential computing, Landlock support and more!

Agenda with conference link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Free to attend, and hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
In just under 21 hours from now, at 11:00 a.m. EDT (UTC-5) on Tues Oct 4, the Podman Community Meeting will be starting up. We'll have demos on Distrobox, the Vault Test Suite, and an open forum for any questions and answers. Free to attend virtually, meeting info is on the agenda: [https://podman.io/community/meeting/agenda/](https://t.co/tEqURz2atf) 

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
A large number of the regular attendees will be involved in another meeting at the same time next week, 
including myself.  Given that and the fact that we currently have no topics for the meeting, we have decided 
to cancel the Podman Community Cabal meeting next Thursday, October 20, 2022.

The next Podman Community Cabal Meeting will be on Thursday, November 17, 2022, at 
11:00 a.m. EST (UTC-4).  Please note that the US will be rolling back our clocks an hour between then 
and now.  If you have any topics that you'd like to discuss, please let me know.

**Comment by TomSweeneyRedHat** [comment]
In just under 24 hours, at 11:00am EST (UTC-5), we'll be holding the Podman Community Cabal Meeting. Currently, there is only one topic about Docker Compose Support, so there is plenty of time for questions or other topics! Conference link in the Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is happening in just over 23 hours on Tuesday, December 6, 2022, at 11:00 am EST(UTC-5). The agenda is here: https://podman.io/community/meeting/agenda/, which includes a link to the video room. We also have a topic to replace the TBD on the agenda, Ygal Blum will be talking about Embedding inside an AutoSD Image, along with our MinIO topic/demo.  Unfortunately, GitHub is not behaving at the moment, and I can't get the agenda page to update at the moment.

**Reply by Luap99** [reply]
@TomSweeneyRedHat This thread is quite long now so we have to scroll a lot to get to the bottom. I would recommend you also always edit your original post at the top to contain the current up to date text. This will make it easier for people to follow IMO.

**Reply by TomSweeneyRedHat** [reply]
Good thought @Luap99 !

**Comment by TomSweeneyRedHat** [comment]
As we have no subjects as of yet, and I have had a number of people tell me they won't be able to attend due to their holidays starting early or other related holiday gatherings going on, I'm going to cancel the Podman Community Cabal Meeting for Thursday, December 15, 2022, at 11:00 a.m. EST (UTC-5).

The next Podman Community Cabal Meeting will be held on Thursday, January 19, 2023, at 11:00 a.m. EST (UTC-5).  The next Podman Community Meeting will be held on Tuesday, February 7, 2023, at 11:00 a.m. EST (UTC-5).  As a reminder, the Cabal meetings are generally to talk about Design questions, futures, and related items in regards to Podman.  The Community meeting is used for announcements, demos, and things of interest not only for Podman but anything container related.

If you have something that you'd like to present at either meeting, please let me know.  Likewise, if you have seen something interesting in the container world that you'd like to hear more about, please let me know, and I'll see if I can find someone to do a presentation on it.

**Comment by TomSweeneyRedHat** [comment]
Podman Community Cabal Meeting - Thursday, January 19, 2023, 11:00 a.m. EST (UTC-5)

The next Podman Community Cabal meeting is happening in just under 20 hours from now, at 11:00 am EST (UTC-5). We'll be talking about Podman v4.4, auto-closing issues, and more! Agenda with video conference link: [https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both](https://t.co/sYgIaGn8rS)

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
Hi All,

The Podman Community Meeting is happening just over 20 hours from now, at 11:00 am EST (UTC-5) tomorrow, Feb 7, 2023. There'll be a demo on Pasta in Podman and updates on Podman v4.4 and Desktop. Video link in the Agenda, hope to see you there! https://podman.io/community/meeting/agenda/

t

**Comment by TomSweeneyRedHat** [comment]
Hi All,

The next Podman Community Cabal Meeting is happening just over 21 hours from now at 
11:00 am EST (UTC-5). We'll be talking about DNS configuration in Podman, and have plenty 
of room for your questions/thoughts! Free to attend, video link in the 
Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

t

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal meeting is happening in just over 21 hours tomorrow, Thursday, March 16, 2023, at 11:00 am EDT (UTC-5).  We'll be talking about SQLite in Podman, Perf Scripts, and Test Days. Agenda with video link:  https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both  Note that the US has gone to Daylight savings, but Europe hasn't.

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting is happening tomorrow, a little over 24 hours from now at 11:00am EDT (UTC-4).  We're talking Database update, Netavark  plugins, and a Desktop demo.  Agenda here: https://podman.io/community/meeting/agenda/, please note the new Google Meeting room listed there.

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal meeting is happening two weeks from right now, on Thur, April 20, 2023, at 11:00 am EDT (UTC-5).  Currently, our agenda is wide open for topics, so if you have a topic to discuss about Podman, please add it to the agenda!  https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal meeting is happening in just over 21 hours, at 11:00 am EDT (UTC-4), Thursday, Apr 20, 2023. We'll be talking Podman v5 features, containersh, and more!  Agenda with video link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

    Hope to see you there,

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal Meeting is happening a week from now, on Thur, May 18, 2023, at 11:00 am EDT (UTC-5). We'll be talking containers, splitting storage layers, and more! Agenda with video link:  https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg  Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The Podman Cabal meeting will be happening in just under 24 hours from now, at 11:00am EDT (UTC-5) on Thu May 18. We will be talking about Storage layers, containersh, [http://podman.io](https://t.co/R5FbTKrfcV) revamp, and the Containers AppStore. Agenda w/ conference link: [https://hackmd.io/gQCfskDuRLm7iO](https://t.co/sYgIaGmACk)

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is just 13 days, 21 hours, and a few minutes from now!  We'll be talking about the ChRIS project's use of Podman, Podman v4.5, a Qudadlet demo, and Podman Desktop v1.0.  Agenda with video link: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w #opensource #podman 

That is Tuesday, June 6, 2023, at 11:00 am EDT (UTC-5).  Hope to see you there!

**Reply by TomSweeneyRedHat** [reply]
Make that UTC-4, not UTC-5 tomorrow...

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting is happening in just under 24 hours at 11:00 am EDT (UTC-5), tomorrow, Tuesday, June 6, 2023.  Topics: ChRIS project demo, Quadlet demo, Podman v4.5 Demo, and Podman Desktop updates.  Video Conf link on the agenda: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w 

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal Meeting will happen in just under a week, on Thursday, Jun 15, 2023, at 11:00 am EDT (UTC-4). We'll be talking about Additional Layer Storage and possible ipfs integration. Conference link in the agenda. If you have Podman design topics to chat about, please join! https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is just hours away! In a little over 17 hours, at 11:00am EDT (UTC-4) on June 15, 2023, we'll be talking about Additional Layer Storage and ipfs integration. Join the fun! Video link and agenda here: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting will be happening next Thursday,  Jul 20, 2023, at 11:00 am EDT (UTC-5). We'll be talking passwd/group handling, ipfs integrations, and more. Please note, we will be using a new Google Meet room. Agenda with room link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

There's room for more topics, if you have one you'd like to discuss, please add it to the agenda.

**Comment by TomSweeneyRedHat** [comment]
The Podman Cabal meeting is happening in just over 23 hours from now at 11:00am EDT (UTC-5), Thu Jul 20, 2023. We'll be talking passwd/group entry handling, ipfs, and more. The new video conference link is in the agenda, hope to see you there! https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting will happen a week from Tuesday, on Tuesday, August 1, 2023, 11:00 am EDT (UTC-4). We have demos on Quadlet and podmansh lined up. We have room for more, so feel free to sign up to talk or add a topic to the agenda: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w video link there too.

**Comment by TomSweeneyRedHat** [comment]
We were down to only one topic for the meeting tomorrow, and the presenter isn't able to make it now due to a scheduling conflict.  Given that and the number of people that are enjoying some holiday time, we have decided to cancel the Podman Community Meeting tomorrow, Tuesday, August 1, 2023, at 11:00 am EDT.  We are still planning to meet on Thursday, August  17, 2023, at 11:00 am EDT for the Cabal meeting, and then on Tuesday, October 3, 2023, at 11:00 am EDT, we will hold the next Podman Community meeting.

If you have topics for either meeting, please send them along to me or add them to the agenda listed below:

Community Meeting: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w

Cabal Meeting: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The next Podman Cabal meeting is just over 6 days away.  We will meet next Thursday, Aug 17, 2023, at 11:00 am EDT (UTC-4). We only have one topic at the moment to discuss, ipfs integration.  Topics can be added to the agenda, and it has the video room URL.  https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both  Hope to see you there!

**Reply by rhatdan** [reply]
I added a couple of topics.

**Reply by TomSweeneyRedHat** [reply]
Due to scheduling issues and having only one topic when we made the decision, we are canceling the Thursday,  Aug 17, 2023, Podman Community Cabal Meeting at 11:00 am EDT. We will meet next on Thursday, Sep 21, 2023. If you have a topic, please add it to the agenda! https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Apologies for the late notice,

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is in just under two weeks.  We'll meet on Thursday, Sept 21, 2023, at 11:00 a.m. EDT (UTC-4).  We currently have a few topics on Podman v4.7, but we would love to get more.  If you have a topic, please add it to the agenda! https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Hope to see you there!

**Reply by TomSweeneyRedHat** [reply]
The Podman Community Cabal Meeting is happening tomorrow in a little over 19 hours from now at 11:00 a.m. EDT (UTC-4) on Thu Sept 21, 2023. We'll discuss Podman v4.7 and have room for more subjects. Agenda with Video URL: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both Hope to see you there!

t

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is happening in just over 5 days from now, on next Tuesday, Oct 3, 2023, at 11:00 am EDT(UTC-4). We'll have a demo on Quadlet, one on Modules, and room for more if you have topics. Agenda with Conference link: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is happening in just under 48 hours on Thu Oct 19, at 11:00am EDT (UTC-4). We'll discuss guest OS specification in the Podman machine and have room for other topics! Agenda with video link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
6 days and 21 hours from now, the next Podman Community Cabal meeting will be happening! We have moved to Tuesdays, and we meet next Tues November 21, 2023, 11:00 am EST (UTC-5). Happy to take topics! Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
Two for one deal!  We currently have no topics for tomorrow's (November 21) Podman Community Cabal meeting, and we have been receiving several notes from people saying they will not be attending due to holiday plans.  We suspect similar issues will occur at the next meeting, which is scheduled for Wednesday, December 19, 2023.

Given that, we have decided to drop tomorrow's meeting and combine the November and December Podman Community Cabal meetings, which will be held on Tuesday, December 12, 2023, at 11:00 a.m. EST (UTC-5).

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting tomorrow, Tuesday, December 5, 2023, at 11:00 am EST (UTC-5) has been canceled due to a lack of topics and cancellations. We are on track to meet at the Cabal meeting on Tuesday, December 12, 2023, at 11:00 am EST. Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there on the 12th!

**Comment by TomSweeneyRedHat** [comment]
There's a Podman Community Cabal Meeting tomorrow, Tues Dec 12, 2023, at 11:00 am EST (UTC-5). We'll be talking about Confidential Containers and Backporting without a Release branch. Agenda with video link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The first Podman Community Cabal meeting of 2024 will start one week from now! We will meet on Tuesday, Jan 16, 2024, at 11:00am EST (UTC-5).

Current talks on krun, crun on qemu, and image signing, and maybe more!
Agenda with conference link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is happening two weeks from today.  Tuesday February 6, 2024, 11:00am EST (UTC-5).  We have one probable topic now and would love to have one or two more!  Agenda with conference link: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w?both Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Meeting is happening in just over one week from now at 11:00 am EST (UTC-4) Tuesday, February 6, 2024. We'll be talking about Podman at Home and more! Agenda with video conference URL: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is happening tomorrow, Feb 20, 2024, at 11:00am EST (UTC-5). We will be talking about podman compose, image/container volumes, and Podman kube to handle VMs. Agenda with video link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
A week from now, the Podman Community Cabal meeting will be in full swing! That will be Tuesday, March 19, 2024, at 11:00 a.m. EDT (UTC-5). It is free to attend. The topics and video link are in the agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both. Note that the US is now on daylight savings.

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is this Tuesday, April 16, 2024, at 11:00 am EDT (UTC-5).  At the moment we only have a topic about backup appliances and would love to have another topic or two!  Agenda with video link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Community Cabal meeting is happening in two weeks, on Tuesday, May 21, 2024, at 11:00 am EDT (UTC-5).  Currently, there are no topics, so if you want to discuss something, please add it to the agenda! https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
In just over 4 days from now, on Tuesday, May 21, 2024, at 11:00 a.m. EDT (UTC-5), we'll be in the Podman Cabal meeting. Currently only one topic about Fedora v5.0 Test Days, and room for more to be added! Agenda with video link: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both #podman #opensource

Hope to see you there!

**Comment by TomSweeneyRedHat** [comment]
The next Podman Cabal meeting will be happening one week from today, on Tuesday, Sept 3, 2024, at 11:00 a.m. EDT (UTC-4). Currently, we only have one topic about issues internal to Red Hat, and happy to take more. Add a topic to the agenda! https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both 

Hope to see you there!

t

**Comment by TomSweeneyRedHat** [comment]
Next week at this time, Tuesday, November 5, 2024, 11:00 am EST (UTC-5), we'll be in the middle of the Podman Cabal meeting! Currently no topics, but we would love to have you add some! Agenda here:

https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
Due to a lack of topics and trying to recover from the holidays, we are canceling the Podman Cabal meeting scheduled for tomorrow, Jan 7, 2025, at 11:00 a.m. EST (UTC-5). The Cabal will meet on Mar 4, 2025, at 11:00 a.m. EST (UTC-5).

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting will happen a week from now at 11:00 am EST (UTC-5).  At the moment, we have plenty of room for topics if you'd like to add one to our agenda!  A nice demo of what you're working on?  A topic you'd like to discuss?  Add it to the agenda!  https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w?both

**Comment by TomSweeneyRedHat** [comment]
In under 21 hours from now, Tues, Mar 4, 2025, at 11:00 AM EST (UTC-5), the Podman Community Cabal meeting will be underway!  We'll discuss CNCF Governance, Better Ways to Build Images, and have room for a topic or two!  Meeting info on the Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
Two weeks from now, on Tuesday, April 1, 2025 at 11:00 am EDT (UTC-4), the Podman Community Meeting will be going on.  We don't have any topics at the moment, so it's a great time to add your demo to our agenda!  Hope to see you there!  https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w?both

**Comment by TomSweeneyRedHat** [comment]
Hey all!  We've an exciting Podman Community Meeting coming up in one week and a few minutes from now on Tues, Aug 6, 11:00 am EDT (UTC-4).  We'll be talking Orches, MCP, and Reproducible Container Builds.    Agenda with meeting info: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w?both #podman #opensource

**Comment by TomSweeneyRedHat** [comment]
A week and a few minutes from now, on Sept 2, 2025, at 11:00 am EDT (UTC-4), the Podman Cabal Meeting will be going on with updates on CNCF, Mono Repo, and Podman 6.0 plans.  Please note we're moving to Zoom!  Topics and the new conference link in the Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Meeting is next Tues, Oct 7, 2025, at 11:00 am EDT (UTC-4). Currently, our agenda is wide open.  Feel free to send suggestions for topics, or add your own to: https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w. The conference link in the agenda, free to attend!

**Comment by TomSweeneyRedHat** [comment]
The Podman Community Cabal Meeting is next Tuesday, November 4, 2025, at 11:00 a.m. EST (UTC-5).  Note, the US is dropping out of Daylight saving time over the weekend.  Also, we are full on with Zoom going forward, the link for that is in the agenda.

We generally discuss design changes here, and the agenda is wide open for now; add a topic and join us! Agenda: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both #podman #opensource

t

**Comment by TomSweeneyRedHat** [comment]
With all the holiday goings on, don't forget the Podman Community meeting tomorrow, Tuesday, December 2, 2025, at 11:00 am EST (UTC-5)!  We'll have a 2025 retro, talk about immutable GitHub releases, Materia, and probably more! Video link in agenda, and room for another topic or two!

Hope to see you there! https://hackmd.io/fc1zraYdS0-klJ2KJcfC7w

**Comment by TomSweeneyRedHat** [comment]
Due to a lack of topics and a number of folks saying they can't make the meeting tomorrow, we're canceling the Tuesday, January 6, 2026, Podman Cabal meeting.  The next Podman Cabal meeting will be on Tuesdau, March 3rd,  and the next Podman Community Meeting will be on Tuesday, Feb 3rd.   Happy New Years!

**Comment by TomSweeneyRedHat** [comment]
On Tuesday, March 3, 2026, at 11:00 a.m. EST (UTC-5), just  6 days from now, we will hold the Podman Cabal meeting. At the moment, the agenda is wide open for your topics! Please add one, and hope to see you there!  https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both
